app/components/dashboard-calendar.js

```diff
@@ -22,8 +21,14 @@ export default Component.extend({
   currentUser: service(),
   store: service(),
   i18n: service(),
+  activeFilters: null,
+  schoolPickedByUser: null,
   selectedDate: null,
   selectedView: null,
+  selectedCohorts: [],
```

Comment:
These should be `null` initially otherwise the Array object can get passed by reference and not cleaned up when the component is removed from the page.